packages/editor/src/lib/exports/getSvgJsx.tsx

```diff
@@ -61,13 +61,18 @@ export function getSvgJsx(editor: Editor, ids: TLShapeId[], opts: TLImageExportO
 	if (opts.bounds) {
 		bbox = opts.bounds
 	} else {
-		for (const { id } of renderingShapes) {
+		for (const { id, shape } of renderingShapes) {
 			const maskedPageBounds = editor.getShapeMaskedPageBounds(id)
 			if (!maskedPageBounds) continue
+			const paddingForShape = typeof padding === 'function' ? padding(shape, editor) : padding
+			const expandedBounds =
+				paddingForShape === 0
+					? maskedPageBounds
+					: maskedPageBounds.clone().expandBy(paddingForShape)
 			if (bbox) {
-				bbox.union(maskedPageBounds)
+				bbox.union(expandedBounds)
 			} else {
-				bbox = maskedPageBounds.clone()
+				bbox = expandedBounds
```

Comment:
isn't this gonna end up with weird stuff happening like if you draw something near the edge of the image it'll add padding to that edge?

<img width="849" height="746" alt="image" src="https://github.com/user-attachments/assets/0f596d72-3943-4d68-9ec2-e9a91c515de8" />